batchconsumer/batchermanager.go

```diff
@@ -100,7 +100,7 @@ func (b *batcherManager) sendBatch(batcher *batcher, tag string) {
 	case PartialSendBatchError:
 		lg.ErrorD("send-batch", kv.M{"msg": e.Error()})
```

Comment:
LGTM, but what's the difference between `lg` and `b.failedLogsFile`? Do they go different places?

Reply:
Yeah, one is written to stderr and the other is written to a file.  The thinking is that when the logging pipeline is going nuts, stderr might not be easily accessible.